zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/impl/TypedEventRegistry.java

```diff
@@ -143,6 +144,7 @@ public final class TypedEventRegistry {
     registry.put(ValueType.ASYNC_REQUEST, AsyncRequestRecord.class);
     registry.put(ValueType.MULTI_INSTANCE, MultiInstanceRecord.class);
     registry.put(ValueType.RUNTIME_INSTRUCTION, RuntimeInstructionRecord.class);
+    registry.put(ValueType.HISTORY_DELETION, HistoryDeletionRecord.class);
```

Comment:
@lenaschoenburg this triggered your review 🙃. Feel free to just approve you don't need to check the rest.